views/js/component/activityMonitoring/currentAssessmentActivity/currentAssessmentActivity.tpl

```diff
@@ -0,0 +1,49 @@
+<div class="col-4 dashboard-block">
```

Comment:
It may be better to wrap all the elements into a root div. This will allow to scope the whole component more efficiently, be it in css or javacript selector.